fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -84,44 +88,44 @@ def messagepull(anchor: str, before: int, after: int, keyword: str) -> Dict:
                 {"operator": "search", "operand": keyword},
             ],
             }
-    result = client.get_messages(request)
+    result = CLIENT.get_messages(request)
     return result 
 
 
-def format_timestamp(i):
-    """Format datetime string"""
-    return datetime.utcfromtimestamp(i).strftime('%Y-%m-%d %H:%M:%S')
-
-
 # TODO: will probably want to pass (i) chat.fhir.org, (ii) list of streams
-def get_data(key):
+def get_data(keyword: str):
```

Comment:
This is just a preference of mine; I tend to prefer longer variable names. It's a bit of a tradeoff.

I also add types to a lot (but not all) of my variables, especially function params. Python will actually ignore this (I'm not sure actually how to turn on strong type checking errors), but IDEs will sometimes highlight warnings in your code if you try to assign/pass something of the wrong data type, so declaring your data types like this can be handy.